blogposts/2021/show-us-your-calendar.md

```diff
@@ -4,151 +4,163 @@ description: 'Engineers are leading the way toward rethinking the 40 hour work w
 author: Nick Moore
 authorUrl: https://twitter.com/nickwritesit
 published: true
-publishDate: 2021-09-16T10:00-07:00
+publishDate: 2021-09-23T10:00-07:00
+tags: [blog]
 slug: remote-work-calendar
 heroImage: https://storage.googleapis.com/sourcegraph-assets/blog/Show%20Us%20Your%20Calendar%20Images/Show%20Us%20Your%20Calendar%20Hero.png
 socialImage: https://storage.googleapis.com/sourcegraph-assets/blog/Show%20Us%20Your%20Calendar%20Images/Show%20Us%20Your%20Calendar%20Hero.png
 ---
 
 ![Flexible work calendar graphic](https://storage.googleapis.com/sourcegraph-assets/blog/Show%20Us%20Your%20Calendar%20Images/Show%20Us%20Your%20Calendar%20Hero.png)
 
-Work is only one part of our lives but until remote work, it occupied a sacrosanct place in our schedules. You were to work between the hours of 9:00 am and 5:00 pm, not including your commute to and from the office, and the rest of your life had to flow around that block.
+Work is only one part of our lives, but until remote work, it occupied a sacrosanct place in our schedules. You were to work between the hours of 9 am and 5 pm, not including your commute to and from the office, and the rest of your life had to flow around that block.
 
 Remote work offers the opportunity to rethink work from first principles and ask ourselves: What does life look like when our jobs work for us, rather than the other way around? When you don’t have to commute, when no one is watching you work, when code shipped at 2 am is as valuable as code shipped at 10 am, what can your life become?
 
 This is [r/battlestations](https://www.reddit.com/r/battlestations/) by way of Google Calendar: When your calendar is a creative space, what will you make?
 
 ## Loïc Guychard, engineering manager, has a four-day work week
 
-[Loïc Guychard](https://about.sourcegraph.com/handbook/company/team#lo%C3%AFc-guychard), engineering manager at Sourcegraph, compares work-time to closet space: “If you have it, you’re going to use it. Whether you use it for things that matter, that’s a different story.”
+[Loïc Guychard](https://about.sourcegraph.com/handbook/company/team#lo%C3%AFc-guychard), engineering manager at Sourcegraph, compares work time to closet space: "If you have it, you're going to use it. Whether you use it for things that matter, that's a different story."
```

Comment:
I got muddled and thought we used straight quotation marks and apostrophes 🤦‍♀️ ... I'm going to change these back! https://about.sourcegraph.com/handbook/communication/content_guidelines/style_and_mechanics#quotation-marks